lib/active_merchant/billing/gateways/kushki.rb

```diff
@@ -24,6 +24,23 @@ def purchase(amount, payment_method, options={})
         end
       end
 
+      def authorize(amount, payment_method, options={})
+        MultiResponse.run() do |r|
+          r.process { tokenize(amount, payment_method, options) }
```

Comment:
Other than tokenization, Kushki only operates on tokens, not directly on payment methods; since this integration doesn't currently support the use of tokens, we tokenize a payment method each time.